packages/javascript/bh-shared-ui/src/components/AssetGroupEdit/AssetGroupEdit.tsx

```diff
@@ -15,27 +15,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Box, Paper } from '@mui/material';
-import { AssetGroup, AssetGroupMemberParams, UpdateAssetGroupSelectorRequest } from 'js-client-library';
+import {
+    AssetGroup,
+    AssetGroupMemberParams,
+    AssetGroupMemberCounts,
+    UpdateAssetGroupSelectorRequest,
+} from 'js-client-library';
 import { FC, useEffect, useState } from 'react';
 import { AssetGroupChangelog, AssetGroupChangelogEntry, ChangelogAction } from './types';
 import AssetGroupAutocomplete from './AssetGroupAutocomplete';
 import { SubHeader } from '../../views/Explore';
-import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { useMutation, useQueryClient } from 'react-query';
 import { apiClient } from '../../utils';
 import AssetGroupChangelogTable from './AssetGroupChangelogTable';
-import {
-    ActiveDirectoryNodeKind,
-    ActiveDirectoryNodeKindToDisplay,
-    AzureNodeKind,
-    AzureNodeKindToDisplay,
-} from '../../graphSchema';
 import { useNotifications } from '../../providers';
 
 const AssetGroupEdit: FC<{
     assetGroup: AssetGroup;
     filter: AssetGroupMemberParams;
-    makeNodeFilterable: (node: ActiveDirectoryNodeKind | AzureNodeKind) => void;
-}> = ({ assetGroup, filter, makeNodeFilterable }) => {
+    memberCounts: AssetGroupMemberCounts | undefined;
+}> = ({ assetGroup, filter, memberCounts }) => {
```

Comment:
Is this intended to remove the filterable logic we added last week?  Looks like this might have been build atop an older commit.

Reply:
This simplifies filtered counts logic it's intended 👍

Reply:
Great question!

In this case its not removing the filtering logic because the new `memberCounts` value replaces the need for `makeNodeFilterable`!